internal/backends/nodejs/grab_test.go

```diff
@@ -1,60 +0,0 @@
-package nodejs
-
-import "testing"
-
-func TestParseFile(t *testing.T) {
-	expected := []string{
-		"fs",
-		"assert",
-		"path",
-		"nan",
-		"buffer",
-		"console",
-		"dns",
-		"child_process",
-	}
-
-	content := `
-import * as fs from 'fs';
-import 'assert';
-import { isAbsolute } from 'path';
-import nan from "nan";
-const foo = require('buffer');
-const foo2 = require("console");
-require("cluster", "crypto");
-require('async_hooks', 'constants');
-require('dns',)
-require(` + "`child_process`" + `)
-
-foo('bar');
-
-fs.doThing();
-
-const foo = ({
-	// this is the reason we switched to tree-sitter
-	foo = () => {},
-}) => void 0;
-
-process.exit(1);
-`
-
-	results := make(chan parseResult)
-	go parseFile([]byte(content), results)
-	result := <-results
-
-	if !result.ok {
-		t.Errorf("Parse failed")
-	}
-
-	if len(result.importPaths) != len(expected) {
-		t.Errorf("Expected %d imports, got %d", len(expected), len(result.importPaths))
-	}
-
-	for i, path := range result.importPaths {
-		if i >= len(expected) {
-			t.Errorf("Unexpected import %s", path)
-		} else if path != expected[i] {
-			t.Errorf("Expected %s, got %s", expected[i], path)
-		}
-	}
-}
```

Comment:
Why are we deleting this test? It's hard for me to see that the integration tests added cover these cases, and isn't it nice to have some unit tests for this feature?

Reply:
the unit tests haven't been running in CI anyways. you're right though, i'll add the missing cases here

Reply:
i'll note that the test is also wrong

```
require(` + "`child_process`" + `)
```

Reply:
forgot to attach

Reply:
it was a golang syntax highlighting issue, the above was valid. fixed!